policyengine_us/variables/gov/states/id/tax/income/deductions/retirement_benefits/id_retirement_benefits_deduction.py

```diff
@@ -0,0 +1,44 @@
+from policyengine_us.model_api import *
+
+
+class id_retirement_benefits_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho retirement benefits deduction"
+    unit = USD
+    reference = [
+        "https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3022a/",
+        "https://tax.idaho.gov/wp-content/uploads/forms/EFO00088/EFO00088_03-01-2023.pdf",
+    ]
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.id.tax.income.deductions.retirement_benefits
+        filing_status = tax_unit("filing_status", period)
+        # Line 8a
+        # Max retirement benefits deduction amount
+        cap = p.cap[filing_status]
+        # Line 8c
+        # Social Security retirement benefits received
+        person = tax_unit.members
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        ss_amount = (
+            person("social_security_retirement", period) * head_or_spouse
+        )
+        total_ss_amt = tax_unit.sum(ss_amount)
```

Comment:
Filter to head or spouse here instead of above. `ss_amount` doesn't indicate that it's zero for dependents as Python variable name.
```suggestion
        total_head_spouse_ss = tax_unit.sum(ss_retirement * head_or_spouse)
```